src/midifile.jl

```diff
@@ -114,13 +114,85 @@ function writeMIDIFile(filename::AbstractString, notes::Notes)
     return midi
 end
 
+"""
+    qpm(midi)
+Return the QPM where the given `MIDIFile` was exported at.
```

Comment:
```suggestion
Return the QPM where the given `MIDIFile` was exported at.
```

You should explain here what the initials QPM stand for.